core/src/main/scala/kafka/controller/ControllerState.scala

```diff
@@ -60,6 +60,8 @@ object ControllerState {
 
   case object AlterPartitionReassignment extends ControllerState {
     def value = 5
+
+    override def rateAndTimeMetricName: Option[String] = Some("PartitionReassignment")
```

Comment:
"RateAndTimeMs" suffix is missing on metricname.

Reply:
Oops. Thanks for catching this :)